### ISSUE TITLE
metrics-Agg typo: imagePullSecret instead of imagePullSecrets in deploymentMetricsAggregator.yaml

### ISSUE THREAD

**Issue by druesendieb** [splunk/splunk-connect-for-kubernetes#429]
**What happened**:
If you use an imagePullSecret for the metricsAgg deployment, helm creates invalid YAML.

Created YAML:
(obtained through `helm template` to analyse)

`splunk-kubernetes-metrics/deploymentMetricsAggregator.yaml`
```
      imagePullSecret:
      - name: SECRETNAME
```
**What you expected to happen**:
Create valid YAML

`splunk-kubernetes-metrics/deploymentMetricsAggregator.yaml`
```
      imagePullSecrets:
      - name: SECRETNAME
```
**How to reproduce it (as minimally and precisely as possible)**:
Used Settings
```
splunk-kubernetes-metrics:
  imageAgg:
(...)
    # Indicates if the image should be pulled using authentication from a secret
    usePullSecret: true
    # The name of the pull secret to attach to the respective serviceaccount used to pull the image
    pullsecretName: SECRETNAME
```

**Anything else we need to know?**:
PullRequest #430  

**Environment**:
SCK: 1.4.2

**Comment by cheneysplunk** [comment]
Good catch! @rockb1017 , would you please review the PR? Thanks.

**Comment by rockb1017** [comment]
Thank you for reporting and creating PR. 
Due to testing pipeline not opened for PRs from forked repo yet, I had to copy your changes and create PR myself. We are working on opening up pipelines soon. 
Fix is released with 1.4.3
Thank you!
